src/data_management/wms_layers/WmsLayer.js

```diff
@@ -292,108 +292,99 @@ class WmsLayer extends Component {
     );
 
     const htmlWmsLayerTableBody = (
-      <div
-        style={{
-          position: "relative"
-        }}
-      >
+      <div className={`${gridStyles.Row}`}>
         <div
-          style={{
-            visibility: this.state.isFetching ? "hidden" : "visible"
-          }}
+          className={`${gridStyles.colLg12} ${gridStyles.colMd12} ${gridStyles.colSm12} ${gridStyles.colXs12}`}
         >
-          <Scrollbars
-            autoHeight
-            autoHeightMin={450}
-            autoHeightMax={450}
-            style={{ width: "100%" }}
-          >
-            {this.state.paginatedWmsLayers.map((wmsLayer, i) => {
-              return (
-                <div 
-                  className={`${wmsLayerTableStyles.tableBody}`}
-                  key={wmsLayer.uuid}
-                >
-                  <div className={`${wmsLayerTableStyles.tableCheckbox}`}>
-                    <input
-                      type="checkbox"
-                      // Make sure that you can still use the checkbox to click on,
-                      // in combination with the check all checkbox.
-                      onClick={this.clickRegularCheckbox}
-                      checked={
-                        this.state.checkboxes[i]
-                          ? this.state.checkboxes[i].checked
-                          : false
-                      }
-                      id={"checkbox_" + i}
-                    />
-                  </div>
-                  <div
-                    className={`${wmsLayerTableStyles.tableName}`}
-                  >
-                    <NavLink
-                      to={`/data_management/wms_layers/${wmsLayer.uuid}`}
-                      style={{
-                        color: "#333"
-                      }}
-                    >
-                      {wmsLayer.name}
-                    </NavLink>
-                  </div>
-                  <div className={`${wmsLayerTableStyles.tableDescription}`}>
-                    <NavLink
-                      to={`/data_management/wms_layers/${wmsLayer.uuid}`}
-                      style={{
-                        color: "#333"
-                      }}
+
+          {total === 0 && isFetching === false ? (
```

Comment:
I see that we have different ways of showing the no results across different management screens. Sometimes, we check for total and isFetching, sometimes we check for only total and in the scenarios, we check for the length. Also, this part is quite similar among all the management screens so maybe we can refactor it into a separate component and reuse it. But I think it would be for another ticket since it can take quite a lot of time as well.

Reply:
I agree, I will make a new ticket for this.